### PR TITLE
Invert function of LOD modifier

### PR DESCRIPTION
**Pull request by DamianMadden**
Resolves [AB#348](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/348)

Also missed some things originally

### CODE REVIEW

File: src/scene/vcLiveFeed.cpp [hunk 1]
```diff
@@ -306,6 +305,8 @@ vcLiveFeed::vcLiveFeed(vdkProject *pProject, vdkProjectNode *pNode, vcState *pPr
   m_oldestFeedUpdate = m_newestFeedUpdate;
   m_fetchNow = true;
 
+  m_labelLODModifier = 1.0;
```
Comment (OsiJr) [inline src/scene/vcLiveFeed.cpp]: This was moved out of the initialiser list?

Comment (DamianMadden) [reply src/scene/vcLiveFeed.cpp]: one of the build machines had a problem with it

Comment (OsiJr) [reply src/scene/vcLiveFeed.cpp]: There is a rule in C++ that the list here has to be in the same order as they are defined in the header; you probably needed to move it above the mutex.

Comment (OsiJr) [reply src/scene/vcLiveFeed.cpp]: Most compilers don't care though

**Pull request merged**